packages/simcore-sdk/src/simcore_sdk/node_data/data_manager.py

```diff
@@ -161,18 +164,22 @@ async def pull(
     *,
     io_log_redirect_cb: LogRedirectCB | None,
     save_to: Path | None = None,
+    r_clone_settings: RCloneSettings | None,
     progress_bar: ProgressBarData,
 ) -> None:
     if file_or_folder.is_file():
-        return await _pull_file(
+        await _pull_file(
             user_id,
             project_id,
             node_uuid,
             file_or_folder,
             save_to=save_to,
             io_log_redirect_cb=io_log_redirect_cb,
+            r_clone_settings=r_clone_settings,
             progress_bar=progress_bar,
         )
+        return

```

Comment:
Same suggestion as above

Reply:
leaving it like this since I will refactor this away